zcash_keys/Cargo.toml

```
@@ -0,0 +1,91 @@
+[package]
+name = "zcash_keys"
+description = "Zcash key and address management"
+version = "0.1.0"
```

```suggestion
version = "0.0.0"
```
until this is in a state we want to publish.